### ISSUE TITLE
Brackets Change Colour On Scroll

### ISSUE THREAD

**Issue by ex-jedi** [jrattray13/swackets#40]
![universal scss c__users_mark_documents_web design local_host_katephoenix atom 04_04_2017 10_12_59](https://cloud.githubusercontent.com/assets/3149496/24651001/0a7103d8-1924-11e7-9cad-e1701df1aadb.gif)

**Comment by AlJohri** [comment]
I just experienced this same issue while using swackets with clojure. I found its related to comments or comment blocks in some way.

**Comment by AlJohri** [comment]
Notice how in the screenshots below, text that is commented out is getting highlighted. I think this contributes to the issue.

Before Scrolling:

![screen shot 2017-04-20 at 10 18 07 pm](https://cloud.githubusercontent.com/assets/2790092/25259969/4c10db4c-2617-11e7-82f7-c2ace9982673.png)

After Scrolling:

![screen shot 2017-04-20 at 10 18 19 pm](https://cloud.githubusercontent.com/assets/2790092/25259979/4fd1becc-2617-11e7-9c38-6a7834dfe9bc.png)

**Comment by Liath** [comment]
If only I could reliably trigger this, it's like coding in a disco!

**Comment by jrattray13** [comment]
Let me know if latest Swackets v0.35 does anything to aide/worsen it, thanks!

**Comment by Nfinished** [comment]
@jrattray13 Definitely still happening.

![image](https://user-images.githubusercontent.com/1719791/30330878-3c3996f6-97a4-11e7-80cd-3c5ace9acf13.png)

**Comment by ex-jedi** [comment]
@jrattray13 Yep, still happening for me too I'm afraid.

![Swackets bug](https://media.giphy.com/media/10k1hkCDID4ltm/giphy.gif)

**Comment by AlJohri** [comment]
It seems like it has to do with comments getting interpreted as actual code and messing with the colors

**Comment by AlJohri** [comment]
@jrattray13 would a file where this reproducibly occurs help?

**Comment by CarlosRGL** [comment]
this bug is still present. Shame because its really nice.

**Comment by CaffeinatedTech** [comment]
Yeah it looks like it is choosing colours at random instead of having set colours per depth level.  Like for instance 1st level green, 2nd level yellow, 3rd level orange etc.

Is it only colouring what is visible, or is it doing the whole file?

**Comment by rolzing** [comment]
Still happening and i also want to take the opportunity to say you, thank you for this package.

**Comment by rzjnzk** [comment]
This is an issue for me as well. Maybe @CaffeinatedTech is onto something, in fact, that is what I suspected before finding this issue.
